stdlib/locale.pyi

```diff
@@ -103,6 +103,15 @@ def atof(string: _str, func: Callable[[_str], float] = ...) -> float: ...
 def atoi(string: _str) -> int: ...
 def str(val: float) -> _str: ...
 
+# native gettext functions
+# https://github.com/python/cpython/blob/f4c03484da59049eb62a9bf7777b963e2267d187/Modules/_localemodule.c#L626
+if sys.platform == "linux" or sys.platform == "darwin":
+    def gettext(__msg: _str) -> _str: ...
+    def dgettext(__domain: _str, __msg: _str) -> _str: ...
+    def dcgettext(__domain: _str, __msg: _str, __category: _str) -> _str: ...
+    def textdomain(__domain: _str | None) -> _str | None: ...
+    def bindtextdomain(__domain: _str, __dir: _str) -> _str | None: ...
+
```

Comment:
Any reason not to add the `bind_textdomain_codeset` function?

Reply:
Added